src/utils/rtlUtils.tsx

```diff
@@ -0,0 +1,262 @@
+import {
+  blur as _blur,
+  change as _change,
+  click as _click,
+  clickAndWait as _clickAndWait,
+  focus as _focus,
+  input as _input,
+  type as _type,
+  typeAndWait as _typeAndWait,
+  wait as _wait,
+  allowAndWaitForAsyncBehavior,
+  RenderResult,
+} from "@homebound/rtl-utils";
+import { prettyDOM } from "@testing-library/react";
+import { fail } from "src/utils/index";
+
+export {
```

Comment:
This is the code from `rtl.tsx` that is explicitly packaged/exposed as `@homebound/beam/rtl` for usage in downstream tests